plugin/src/main/java/io/jenkins/plugins/casc/SchemaGeneration.java

```diff
@@ -193,9 +197,21 @@ private static void generateEnumAttributeSchema(JSONObject attributeSchemaTempla
             }
             attributeSchemaTemplate.put(attribute.getName(),
                 new JSONObject()
+                    .put("description", retrieveDocStringFromAttribute(baseConfigurator.getTarget(), attribute.name))
                     .put("type", "string")
                     .put("enum", new JSONArray(attributeList)));
         }
     }
+
+    private static String retrieveDocStringFromAttribute(Class baseConfigclass, String attributeName) {
+        String htmlDocString = null;
+        try {
+            htmlDocString = ConfigurationAsCode.get().getHtmlHelp(baseConfigclass, attributeName);
+        } catch (IOException e) {
+            LOGGER.warning("Error getting help document for attribute : " + e);
+        }
+        htmlDocString = htmlDocString.replaceAll("\\<.*?\\>", "").trim();
```

Comment:
did you try not removing html first to see whether vs code supports it?

Reply:
this should probably be extracted and unit tested as well

Reply:
I did not try it out, but removing it would ensure the schema stays clean

Reply:
worth trying first, just so we know

Reply:
Naah VsCode does not support it :(
I will add a unit test for it :)